code/game/gamemodes/game_mode.dm

```diff
@@ -120,15 +163,39 @@
 		replacementmode.make_antag_chance(character)
 	return
 
+/datum/game_mode/proc/make_special_antag_chance(mob/living/character)
+	if(character.mind.antag_datums)
+		if(character.mind.antag_datums.len > 0)

```

Comment:
Isn't this check redundant?

Reply:
Would if(character.mind.antag_datums) return FALSE if the list is empty but created?

Reply:
Yes

Reply:
Alright, that check can be compressed then